pkg/policy/snapshots.go

```diff
@@ -0,0 +1,228 @@
+package policy
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/jeremywohl/flatten"
+)
+
+func (ce *Executor) StoreSnapshot(ctx context.Context, path string, tables []string) error {
+	if len(tables) == 0 {
+		return errors.New("no tables to snapshot")
+	}
+
+	for _, table := range tables {
+		ef, err := os.OpenFile(fmt.Sprintf("%s/table_%s.csv", path, table), os.O_CREATE|os.O_WRONLY, 0777)
```

Comment:
`filepath.Join` and `afero.Afero` stuff :)

Reply:
Addressed the filepath, have not handled the afero